src/meta/src/hummock/manager/mod.rs

```diff
@@ -115,10 +115,10 @@ pub type HummockManagerRef<S> = Arc<HummockManager<S>>;
 /// by the way
 /// After called, the `ValTransaction` will be dropped.
 macro_rules! commit_multi_var {
-    ($hummock_mgr:expr, $context_id:expr, $($val_txn:expr),*) => {
+    ($hummock_mgr:expr, $context_id:expr, $trx_extern_part:expr, $($val_txn:expr),*) => {
```

Comment:
After adding the extra expr in the macro, we need to pass None when calling `commit_multi_var!` in many places. This hurts readibility. Can we pass `Transaction::default()` explicitly instead if we don't need to reuse a txn?

Reply:
> After adding the extra expr in the macro, we need to pass None when calling `commit_multi_var!` in many places. This hurts readibility. Can we pass `Transaction::default()` explicitly instead if we don't need to reuse a txn?

fixed